resources/artifact-ignores.properties

```
@@ -256,6 +256,29 @@ kubernetes-pipeline-aggregator-1.3
 active-directory-2.1     		    	 # JENKINS-42739
 agc-app-upload-1.0				 # https://github.com/jenkins-infra/helpdesk/issues/2853
 BlameSubversion-1.121    		    	 # big number but not the latest release
+# blueocean accidental deploy
+blueocean-parent-1.26.1-JDK11TEST
+blueocean-1.26.1-JDK11TEST
+blueocean-bitbucket-pipeline-1.26.1-JDK11TEST
+blueocean-commons-1.26.1-JDK11TEST
+blueocean-config-1.26.1-JDK11TEST
+blueocean-core-js-1.26.1-JDK11TEST
+blueocean-dashboard-1.26.1-JDK11TEST
+blueocean-events-1.26.1-JDK11TEST
+blueocean-executor-info-1.26.1-JDK11TEST
+blueocean-git-pipeline-1.26.1-JDK11TEST
+blueocean-github-pipeline-1.26.1-JDK11TEST
+blueocean-i18n-1.26.1-JDK11TEST
+blueocean-jira-1.26.1-JDK11TEST
+blueocean-jwt-1.26.1-JDK11TEST
+blueocean-personalization-1.26.1-JDK11TEST
+blueocean-pipeline-api-impl-1.26.1-JDK11TEST
+blueocean-pipeline-editor-1.26.1-JDK11TEST
+blueocean-pipeline-scm-api-1.26.1-JDK11TEST
+blueocean-rest-1.26.1-JDK11TEST
+blueocean-rest-impl-1.26.1-JDK11TEST
+blueocean-web-1.26.1-JDK11TEST
+jenkins-design-language-1.26.1-JDK11TEST
```

`-parent` isn't a plugin. The rest doesn't exist.

```suggestion
blueocean-commons-1.26.1-JDK11TEST
blueocean-core-js-1.26.1-JDK11TEST
blueocean-i18n-1.26.1-JDK11TEST
blueocean-pipeline-scm-api-1.26.1-JDK11TEST
blueocean-rest-1.26.1-JDK11TEST
blueocean-web-1.26.1-JDK11TEST
jenkins-design-language-1.26.1-JDK11TEST
```